src/csi/backend/plugin/fusionstorage-san.go

```diff
@@ -158,6 +158,20 @@ func (p *FusionStorageSanPlugin) StageVolume(name string, parameters map[string]
 		return err
 	}
 
+	// If the request to stage is for volumeDevice of type Block and the devicePath
+	// is provided then do not format and create FS and mount it.
+	// Simply create a symlink to the devpath on the staging area
+	if parameters["volumeMode"].(string) == "Block" {
```

Comment:
It is dangerous to use assertion without assertion test. IMO, the code can be improved to : 
```if volMode, ok := parameters["volumeMode"].(string); ok &&  volMode == "Block" ```

Reply:
Done. Thanks for pointing the type assertions